combinations/hash.tsv

```diff
@@ -456,3 +456,6 @@ bedtools=2.31.0,samtools=1.17,ucsc-wigtobigwig=447
 r-base=4.0.2,vcftools=0.1.14-0
 perl==5.26.2,perl-sort-key=1.33,perl-yaml-libyaml=0.88
 tophat=2.1.1,samtools=1.17
+perl==5.26.2,r==3.6.0,bcftools==1.13,bedtools==2.30.0,bc==1.07.1
+python==3.7.9,pandas==1.3.4,numpy==1.21.4,pyranges==0.0.115,fire==0.4.0
```

Comment:
you do not need to specify python here, it comes automatically with pandas and numpy, also below